test/resource_parsing.coffee

```diff
@@ -207,3 +208,27 @@ describe 'Resource Parsing', ->
 
 		it 'rhr should be $0', ->
 			assert.equal(filter[2].bind, 0)
+
+	resourceTest '$select=', (result) ->
+		it 'should fail because it is invalid', ->
+			assert(result instanceof SyntaxError)
```

Comment:
It would worth checking for a specific error message, unless it's that generic "syntax error: match failed" in which case ignore me as it's a larger issue unrelated to the pr

Reply:
Yeah, it's the generic one